src/buildercore/bootstrap.py

```diff
@@ -172,16 +172,25 @@ def update_sqs_stack(stackname, **kwargs):
     context = context_handler.load_context(stackname)
     setup_sqs(stackname, context.get('sqs', {}), pdata['aws']['region'])
 
-def unsub_sqs(stackname, context_sqs, region, dry_run=False):
+def unsub_sqs(stackname, cfn_sqs, region, dry_run=False):
     sublist = core.all_sns_subscriptions(region, stackname)
-    struct = {}
-    for queue_name in context_sqs:
-        subscriptions = context_sqs[queue_name] # ll: [bus-articles--prod, ...]
-        struct[queue_name] = [sub for sub in sublist if sub['Topic'] not in subscriptions]
+    prj_sqs = project_data_for_stackname(stackname, interpolate=True)['aws']['sqs']
```

Comment:
Why don't we have everything into `context` to do unsubscriptions? This seems like a layer violation as it shouldn't be needed to recompute project data at this level.

Reply:
(and afaik it also may not substitute `{instance}` with `prod`, and similar